### PR TITLE
Outline for identification of major test changes.

### PR DESCRIPTION
**Pull request by ginjabenjamin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
...
+## Test Changes
+This outline will include proposed test changes that need to be incorporated into OTG v5. These should be proposed significant changes that are associated with an explicit test.
+
+### New Tests
+
+Server-Side Template Injection
+
+### Test Changes
+(Briefly describe which tests need major re-writing)
+
```
Comment (talargoni) [inline README.md]: Testing for Horizontal Bypassing Authorization Schema

**Comment by kingthorin** [comment]
@matowasp do you want these changes integrated into the current README? If so let’s close this one and I’ll open another PR which builds on the current content? (I can ensure that the git author info is preserved.)

**Comment by kingthorin** [comment]
This PR can be closed, it's been superseded by https://github.com/OWASP/OWASP-Testing-Guide-v5/pull/24 which has been approved and merged (and did maintain @ginjabenjamin as the author).

**Comment by kingthorin** [comment]
Closing, adapted in other PRs.

**Pull request closed without merging**